### ISSUE TITLE
Add license-expression library

### ISSUE THREAD

**Issue by steven-esser** [nexB/deltacode#91]
To support #90, we should add the https://github.com/nexB/license-expression library to DeltaCode and update our algorithm to use this library.

**Comment by pratik0316** [comment]
@majurg , @johnmhoran could you explain a bit more about the context of this issue , In what terms will we use this library ?

**Comment by johnmhoran** [comment]
Hi @Pratikrocks .  The scan files produced by https://github.com/nexB/scancode-toolkit contain a number of fields, including `license_expression`.  Our goal is to find a way for DeltaCode -- which compares two ScanCode-Toolkit result files -- to compare and score pairs of simplified license expression values, and we'll need the https://github.com/nexB/license-expression library to do more detailed comparisons.

**Comment by pratik0316** [comment]
Thanks @johnmhoran for the explanation

**Comment by pratik0316** [comment]
I was thinking to integrate `license-expresiion library` to deltacode in the following manner::

> We would provide an option for the user in the command line to specify a license type (eg:MIT..,may be more than one),then based upon the choice we would show the deltas/print in json(or csv) file where either my old or new file has the license(s) specified by the user.

> Moreover we could also provide another command line arguments to display if the deltas when explicitly new path has a license.

> Actually in this way we can Implement many different queries based upon the license detection 

I was planning to integrate this way .What are your views @majurg , @johnmhoran regarding this

**Comment by steven-esser** [comment]
No, we just want the library for handling license_expression strings.

**Comment by pratik0316** [comment]
okay @majurg ,this will be effective in detecting the licenses more proficiently, moreover some redundant methods could be reduced.
